openraft/src/core/raft_core.rs

```diff
@@ -1070,21 +1076,30 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> RaftCore<C,
     /// Spawn a new replication stream returning its replication state handle.
     #[tracing::instrument(level = "debug", skip(self))]
     #[allow(clippy::type_complexity)]
-    pub(crate) async fn spawn_replication_stream(&mut self, target: C::NodeId) -> ReplicationStream<C::NodeId> {
+    pub(crate) async fn spawn_replication_stream(&mut self, target: C::NodeId) -> Option<ReplicationStream<C::NodeId>> {
         let target_node = self.engine.state.membership_state.effective.get_node(&target);
+        let network = match self.network.connect(target, target_node).await {
+            Ok(n) => n,
+            Err(e) => {
+                if tracing::enabled!(Level::ERROR) {
+                    tracing::error!({target=%target}, "failed to connect: {}", e);
+                }
```

Comment:
`if tracing::enabled!(Level::ERROR)` is unnecessary.
`tracing::error!` will not evaluate anything if debug level is not ERROR.